test/pom.xml

```diff
@@ -273,5 +275,41 @@
         </dependency>
       </dependencies>
     </profile>
+    <profile>
+      <id>minicluster-unit-tests</id>
+      <properties>
+        <accumulo.skipMiniClusterInFailsafe>true</accumulo.skipMiniClusterInFailsafe>
+      </properties>
+      <build>
+        <pluginManagement>
+          <plugins>
+            <plugin>
+              <groupId>org.apache.maven.plugins</groupId>
+              <artifactId>maven-surefire-plugin</artifactId>
+              <executions>
+                <!-- Adds an execution for the minicluster-only tests at the test phase -->
+                <execution>
+                  <id>run-minicluster-unit-tests</id>
+                  <goals>
+                    <goal>test</goal>
+                  </goals>
+                  <phase>test</phase>
+                  <configuration>
+                    <groups>${accumulo.miniclusterTests}</groups>
+                    <!-- Includes still limit the search-space for groups -->
+                    <includes>
+                      <include>**/Test*.java</include>
+                      <include>**/*Test.java</include>
+                      <include>**/*TestCase.java</include>
+                      <include>**/*IT.java</include>
```

Comment:
So, just bind another execution of failsafe to `<phase>test</phase>` instead of using surefire? Easy enough.